### DISCUSSION TITLE
osu! Lazer mobile crashing

### DISCUSSION THREAD

**Discussion by christhesilly** [ppy/osu#21443] [Q&A]
### Type

Crash to desktop

### Bug description

When I play osu! Lazer on my tablet (Android) a lot of the time it will crash, even when trying to play a beatmap. 

### Version

2022.1123.0 Lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/10099168/performance.log)
[performance-draw.log](https://github.com/ppy/osu/files/10099169/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/10099170/performance-input.log)
[runtime.log](https://github.com/ppy/osu/files/10099171/runtime.log)
[database.log](https://github.com/ppy/osu/files/10099172/database.log)
[network.log](https://github.com/ppy/osu/files/10099173/network.log)
[performance-audio.log](https://github.com/ppy/osu/files/10099174/performance-audio.log)
[performance-update.log](https://github.com/ppy/osu/files/10099175/performance-update.log)

**Comment by bdach** [comment]
There is no trace of a crash in the log attached. Please make sure you're not restarting the game between reproducing the crash and uploading logs.

**Comment by creativetnt13** [comment]
Idk I have the same problem in my phone

**Comment by OpenSauce04** [comment]
I've actually had this happen recently as well, also on mobile
